vllm/model_executor/models/fuyu.py

```diff
@@ -246,9 +245,8 @@ def __init__(self, vllm_config: VllmConfig, prefix: str = "") -> None:
             quant_config=quant_config,
             gather_output=True,
         )
-        self.language_model = PersimmonForCausalLM(config.text_config,
-                                                   cache_config=cache_config,
-                                                   quant_config=quant_config)
+        self.language_model = PersimmonForCausalLM(
```

Comment:
it would be better if this can use `init_vllm_registered_model` .

I think this `config.text_config` misses an `architectures` field. we can manually add the field, and then call `init_vllm_registered_model`.

we also need to change the signature of `init_vllm_registered_model` in the future.

Reply:
It has `model_type` defined so it should still be possible to match if we add this model to the registry.